src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.service.ts

```diff
@@ -58,12 +56,12 @@ export class DotEditLayoutGridService {
                 .sortBy('config.col')
                 .groupBy('config.row')
                 .values()
-                .map(this.convertToLayoutRow)
+                .map(this.getLayoutRowFromLayoutGridBoxes)
                 .value()
         };
     }
 
-    private convertToLayoutRow(gridBoxes: DotLayoutGridBox[]): DotLayoutRow {
+    private getLayoutRowFromLayoutGridBoxes(gridBoxes: DotLayoutGridBox[]): DotLayoutRow {
```

Comment:
Typical Strict Languages don't require method names to be verbs but is really really recommended.